include/magic_enum.hpp

```diff
@@ -545,7 +547,7 @@ constexpr auto entries(std::index_sequence<I...>) noexcept {
 }
 
 template <typename E>
-inline constexpr auto entries_v = entries<E>(std::make_index_sequence<count_v<E>>{});
+inline constexpr std::array entries_v = entries<E>(std::make_index_sequence<count_v<E>>{});
```

Comment:
I tried to compile only the examples and the tests. It runs without the `names_v` resolution. Probably not used directly anywhere.

But of course, it can be this too.